src/searchForm/SearchForm.tsx

```diff
@@ -212,6 +220,21 @@ export const SearchForm = ({
     formValues[SearchFormElements.AUTOCOMPLETE_FROM] != null ||
     formValues[SearchFormElements.AUTOCOMPLETE_TO] != null
 
+  const getPlaceholderText = (
```

Comment:
It looks like this function can be static.

Reply:
Not sure what you have in mind

Reply:
You can move it out of the component as it doesn't depend on props so you don't recreate it on each render 😉.

Reply:
Oh sorry, sure I moved it